sdk/typescript/src/keypairs/passkey/keypair.ts

```diff
@@ -124,10 +139,11 @@ export class PasskeyKeypair extends Signer {
      * If there are existing passkey wallet, use `signAndRecover` to identify the correct
      * public key and then initialize the instance. See usage in `signAndRecover`.
      */
-    constructor(publicKey: Uint8Array, provider: PasskeyProvider) {
+    constructor(publicKey: Uint8Array, provider: PasskeyProvider, credentialId?: Uint8Array) {
```

Comment:
Why is it marked as optional? In what situation can it be undefined? I have been going through the code and I can't just find when that can happen.
```suggestion
    constructor(publicKey: Uint8Array, provider: PasskeyProvider, credentialId: Uint8Array) {
```

Reply:
that being optional is spread through the whole code too

Reply:
I made it optional because it's not really necessary to make a `PasskeyKeypair`, it just simplifies the authorization request. Since this is in the sdk and can be used in many cases I think it's better to let people decide when they need it or not. In our case (wallet) it is always used for convenience